analyzer-comments/java/annalyns-infiltration/avoid_if_statements.md

```diff
@@ -0,0 +1,3 @@
+# Avoid If statements or ternary operators
+
+This exercise aims to explore booleans and boolean operators. It can be solved without if statements and ternary operators.
```

Comment:
Bit of nit-picking here: the [Exercism docs](https://exercism.org/docs/building/markdown/markdown#h-one-sentence-per-line) dictate to use a single sentence per line, so move the second sentence to the next line.